### PR TITLE
[Resolve #40]separate Web and Console from QC

### LINKED ISSUE #40: separate Web and Console from QC
Web and Console have independent interactive UI.

### PR DESCRIPTION
**Pull request by forchain**
#40

### CODE REVIEW

File: commands/commands.go [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+				return
+			}
+			return
+		}
+	}
+
+	fileCfg.ConfigFile = preCfg.ConfigFile
+
+	if cmd.Flag("debuglevel").Changed {
+		fileCfg.DebugLevel = preCfg.DebugLevel
```
Comment (dindinw) [inline commands/commands.go:85]: The `fileCfg` and `preCfg` look like incorrect namings or lead to confusion. But we can improve/refactor later and let this RP go first.

File: commands/commands.go [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+		fileCfg.Confirmations = preCfg.Confirmations
+	}
+	if cmd.Flag("qtlsskipverify").Changed {
+		fileCfg.QTLSSkipVerify = preCfg.QTLSSkipVerify
+	}
+	config.ActiveNet = utils.GetNetParams(fileCfg.Network)
+	funcName := "LoadConfig"
+	// Parse, validate, and set debug log level(s).
+	if err := parseAndSetDebugLevels(fileCfg.DebugLevel); err != nil {
+		err := fmt.Errorf("%s: %v", funcName, err.Error())
```
Comment (dindinw) [inline commands/commands.go:142]: `funcName` looks only used here.

**Review by dindinw — APPROVED** [review]

File: commands/commands.go (hunk 1 above)
Comment (forchain) [reply commands/commands.go:85]: this function will be substituted by viper,  #43

File: commands/commands.go (hunk 2 above)
Comment (forchain) [reply commands/commands.go:142]: this function will be substituted by viper, #43

**Pull request merged**